specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -119,6 +126,74 @@ export const notificationRenderers: IR<
       />
     );
   },
+  'record-merge-starting'(notification) {
+    const tableName = notification.payload.table;
+    const collectionId = parseInt(notification.payload.collection_id);
+    const mergeName = notification.payload.name;
+    const collection = userInformation.availableCollections.find(
+      ({ id }) => id === collectionId
+    );
+    const [collectionName, setCollectionName] = React.useState<
+      string | null | undefined
+    >('');
+
+    React.useEffect(() => {
+      setCollectionName(collection?.collectionName);
+    }, [collection]);
+
+    return (
+      <>
+        {mergingText.mergingHasStarted()}
+        <div className="flex items-center gap-2">
+          <TableIcon label name={tableName} />
+          <p>
+            {collectionName} {'-'} {mergeName}
+          </p>
+        </div>
+      </>
+    );
+  },
+  'record-merge-failed'(notification) {
+    const tableName = notification.payload.table;
+    const id = parseInt(notification.payload.new_record_id);
+    const ids = [JSON.parse(notification.payload.old_record_ids), id];
+    return (
+      <>
+        {mergingText.mergingHasFailed()}
+        <div className="flex items-center gap-2">
+          <TableIcon label name={tableName} />
+          <Link.NewTab
+            href={formatUrl(`/specify/overlay/merge/${tableName}/`, {
```

Comment:
(optional) this is a bit too complex. for readability, can you extract it into a variable, and just use that variable here?

that will make code more readable